src/viur/core/skeleton.py

```diff
@@ -1408,9 +1408,13 @@ def processRemovedRelations(removedKey, cursor=None):
         .filter("viur_relational_consistency >", 2)
     updateListQuery = updateListQuery.setCursor(cursor)
     updateList = updateListQuery.run(limit=5)
+
     for entry in updateList:
         skel = skeletonByKind(entry["viur_src_kind"])()
-        assert skel.fromDB(entry["src"].key)
+        if not skel.fromDB(entry["src"].key):
+            logging.error(f"processRemovedRelations detects inconsistency on src={entry['src'].key!r}")
+            raise ValueError()
```

Comment:
Can you add the message to the `ValueError` exception too?

Reply:
Fixed!